### DISCUSSION TITLE
first-bug-down/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#39] [Announcements]
# first-bug-down/

Slow Start

https://www.supergoodcode.com/first-bug-down/

**Comment by qwertychouskie** [comment]
Ahem, you rate of blog post production is... lacking.  Even The Chair is still subject to the demands of the people, and the people demand more blog posts.

**Comment by da2ce7** [comment]
Using timelines as a way of tracking work sounds like a real nice cleanup.

**Comment by qwertychouskie** [comment]
The date is Feb 14, 2024.  It has been 37 days since the people last received sustenance.  Has the Chair forsaken us?

**Reply by zmike** [reply]
I was trapped in a time-dilated space and no one sent help
